research-institute-for-nature-and-forest.csl

```diff
@@ -7,63 +7,95 @@
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+      <term name="edition" form="short">edn.</term>
+      <term name="no date" form="short">s.d.</term>
     </terms>
   </locale>
   <locale xml:lang="nl">
     <terms>
       <term name="in">In</term>
       <term name="et-al">et al.</term>
+      <term name="editor" form="short">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+      <term name="edition" form="short">ed.</term>
+      <term name="no date" form="short">s.d.</term>
     </terms>
   </locale>
   <macro name="container">
     <choose>
-      <if type="chapter paper-conference" match="any">
-        <text term="in" prefix=". " suffix=": "/>
-        <names variable="editor translator" delimiter=", " suffix=". ">
-          <name delimiter-precedes-last="always" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
-          <label text-case="lowercase" prefix=" (" suffix=")"/>
-        </names>
-        <group delimiter=", ">
-          <text variable="container-title"/>
-          <text variable="collection-title"/>
-        </group>
+      <if type="article-journal article-magazine article-newspaper" match="none">
+        <choose>
+          <if type="paper-conference" variable="container-title" match="any">
+            <text term="in" suffix=": "/>
+            <names variable="editor translator" delimiter=", " suffix=". ">
+              <name and="symbol" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="all" sort-separator=" "/>
+              <label text-case="lowercase" form="short" prefix=" (" suffix=")"/>
+            </names>
+          </if>
+        </choose>
       </if>
-      <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
-        <group prefix=". " delimiter=", ">
-          <text variable="container-title"/>
-          <text variable="collection-title"/>
-        </group>
-      </else-if>
-      <else>
-        <group prefix=". " delimiter=", ">
-          <text variable="container-title"/>
-          <text variable="collection-title"/>
-        </group>
-      </else>
     </choose>
+    <group delimiter=", ">
+      <text variable="container-title"/>
+      <text variable="collection-title"/>
+      <choose>
+        <if type="article-journal article-magazine article-newspaper" match="any">
+            <!-- show nothing -->
+        </if>
+        <else-if variable="volume">
+          <group delimiter=" ">
+            <text term="volume" form="short" text-case="capitalize-first"/>
+            <number variable="volume"/>
+          </group>
+        </else-if>
+        <else-if variable="collection-number">
+          <group delimiter=" ">
+            <text term="issue" form="short" text-case="capitalize-first"/>
+            <number variable="collection-number"/>
+          </group>
+        </else-if>
+        <else-if variable="number">
```

Comment:
indenting looks wrong here. Could you run the whole style through formatter.citationstyles.org/ please?

Reply:
Fixed this one in a8e945c80. Other formatter updates in 5bdc63c7d, ce08f7f6b, a22ad4bc7